pages/api/csv-export.js

```diff
@@ -68,12 +69,19 @@ export default async function handler(req, res) {
     }
 
     // prefixing question text with Q_
-    acc[vote.participantId][`Q_${statements[vote.statementId]}`] =
+    acc[vote.participantId][`q_${statements[vote.statementId]}`] =
       vote.vote;
 
     return acc;
   }, {});
 
-  // only sending the array (values), not the object
-  res.status(200).send(Object.values(participants));
+  const csv = await new ObjectsToCsv(Object.values(participants));
+
+  res.setHeader('Content-Type', 'text/csv');
+  res.setHeader(
+    'Content-Disposition',
+    `attachment; filename=JamsCSV-${jamId}.csv`,
```

Comment:
A timestamp might be good here too?

Reply:
Actually, should we use the Jam's `name` here as well? Some kind of hyphen-separated-truncated string to make sure it doesn't ge ttoo long and remains vaild?

Reply:
I was thinking when the CSV is generated, as the moderator will then be able to tell which CSV is the most recent.

Reply:
Removed the doc id and replaced with Jam name and date and time of download for the filename of the CSV file.